[Term Entry] Swift Arrays .randomElement() #2561 (#2610)
Codecademy/docs: content/swift/concepts/arrays/terms/randomElement/randomElement.md

---
Title: '.randomElement()'

> Description: 'Returns a random element from of arrary.'

Reviewer: ```suggestion
Description: 'Returns a random element from an array.'
```

Subjects:
  - 'iOS'
  - 'Mobile Development'
Tags:
  - 'Arrays'
  - 'Methods'
CatalogContent:
  - 'learn-swift'
  - 'paths/build-ios-apps-with-swiftui'
---

> The ** `.randomElement()` method will randomly return an element from an  array. If the array is empty, it will return nil.

Reviewer: ```suggestion
The ** `.randomElement()`** method will randomly return an element from an array. If the array is empty, it will return `nil`.
```

## Syntax

```swift 
arrayName.randomElement()

> 

Reviewer: ```suggestion
```

```

> 

Reviewer: Generally we try to include a sentence or two after the syntax block explaining the parameters, or functionality of the method, etc

## Example

```swift
var discount = ["10%", "20%", "30%", "40%", "50%"]
let randomDiscount = discount.randomElement()!
let randomDiscount2 = discount.randomElement()!
print(randomDiscount)
print(randomDiscount2)
```

> The output will be:

Reviewer: ```suggestion
The output will be similar to:
```
